internal/goldendataset/pict_metrics_input_defs.go

```diff
@@ -38,7 +38,6 @@ const (
 	MetricTypeDoubleGauge           PICTMetricDataType = "DoubleGauge"
 	MetricTypeMonotonicDoubleSum    PICTMetricDataType = "MonotonicDoubleSum"
 	MetricTypeNonMonotonicDoubleSum PICTMetricDataType = "NonMonotonicDoubleSum"
-	MetricTypeIntHistogram          PICTMetricDataType = "IntHistogram"
 	MetricTypeHistogram             PICTMetricDataType = "Histogram"
```

Comment:
For the goldendataset should we keep both versions? One would have Double values in the Exemplars the other will have Int Values? Maybe name them "DoubleExemplarsHistogram" and "IntExemplarsHistogram"?

I think is important to test with both possibilities. What do you think?

Right now because we don't have that feature on the pdata.Exemplar implemented to get/set the int value it is ok to generate with "double" exemplar as a work around.

Reply:
This makes sense to me. Will fix this PR.

Reply:
:)